README.md

```diff
@@ -1107,6 +1107,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Vittorio Rinaldo](https://github.com/rinaldovittorio)
   - [Vivek Jaiswar](https://github.com/Narcos101)
   - [Vivek Singh](https://github.com/vivmost)
+  - [Vivek Nakkana](https://github.com/VivekNakkana)
```

Comment:
```suggestion
  - [Vivek Nakkana](https://github.com/VivekNakkana)
  - [Vivek Singh](https://github.com/vivmost)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix